azure-devops/projects/pagopa-projects/tlscert-api-config-dev-platform-pagopa-it.tf

```
@@ -0,0 +1,74 @@
+variable "tlscert-api-config-dev-platform-pagopa-it" {
+  default = {
+    repository = {
+      organization   = "pagopa"
+      name           = "le-azure-acme-tiny"
+      branch_name    = "master"
+      pipelines_path = "."
+    }
+    pipeline = {
+      enable_tls_cert         = true
+      path                    = "TLS-Certificates\\DEV"
+      dns_record_name         = ""
+      dns_zone_name           = "apiconfig.dev.platform.pagopa.it"
+      dns_zone_resource_group = "pagopa-d-vnet-rg"
+      # common variables to all pipelines
+      variables = {
+        CERT_NAME_EXPIRE_SECONDS = "2592000" #30 days
+        KEY_VAULT_NAME           = "pagopa-d-kv"
+      }
+      # common secret variables to all pipelines
+      variables_secret = {
+      }
+    }
+  }
+}
+
+locals {
+  tlscert-api-config-dev-platform-pagopa-it = {
+    tenant_id         = module.secrets.values["PAGOPAIT-TENANTID"].value
+    subscription_name = "DEV-PAGOPA"
+    subscription_id   = module.secrets.values["PAGOPAIT-DEV-PAGOPA-SUBSCRIPTION-ID"].value
+  }
+  tlscert-api-config-dev-platform-pagopa-it-variables = {
+    KEY_VAULT_SERVICE_CONNECTION = module.DEV-PAGOPA-TLS-CERT-SERVICE-CONN.service_endpoint_name
+  }
+  tlscert-api-config-dev-platform-pagopa-it-variables_secret = {
+  }
+}
+
+module "tlscert-apiconfig-dev-platform-pagopa-it-cert_az" {
+  source = "git::https://github.com/pagopa/azuredevops-tf-modules.git//azuredevops_build_definition_tls_cert?ref=v2.0.2"
```

```suggestion
  source = "git::https://github.com/pagopa/azuredevops-tf-modules.git//azuredevops_build_definition_tls_cert?ref=v2.0.1"
```
please use v2.0.1 version, v2.0.2 is in preview and needs others changes (we'll schedule migration on other pr)